swap/src/bin/asb.rs

```diff
@@ -156,10 +157,19 @@ async fn main() -> Result<()> {
             for listen in config.network.listen {
                 Swarm::listen_on(&mut swarm, listen.clone())
                     .with_context(|| format!("Failed to listen on network interface {}", listen))?;
+                Swarm::add_external_address(&mut swarm, listen.clone(), AddressScore::Infinite)
+                    .with_context(|| format!("Failed to add external address {}", listen))?;
```

Comment:
The address we are adding here should come from either a config or a commandline parameter.